packages/metal/src/coreNamed.js

```diff
@@ -289,3 +289,48 @@ export function isString(val) {
  */
 export function nullFunction() {
 }
+
+/**
+ * Register a custom element for a given metal component.
+ *
+ * @param {String} tagName The tag name to use for this custom element.
+ * @param {!function()} ctor Metal component constructor.
+ * @param {?Array=} observedAttrs The attributes to be "observed".
+ * @return {void} Nothing.
+ */
+export function defineWebComponent(tagName, ctor, observedAttrs) {
```

Comment:
Actually, when using 

```javascript
const observedAttrs = State.getStateStatic(Ctor);
```

To get the attributes we want to observe for the component definition, an error is thrown with the following message

```
Failed to execute 'define' on 'Custom Element Registry': Iterator getter is not callable.
```

I'm still trying to find a solution, but that's the reason why I added observedAttrs as an argument to the function.